src/components/LeftSidebar/SearchBox/SearchBox.vue

```diff
@@ -66,6 +66,11 @@ export default {
 		/**
 		 * If true, this component displays an 'x' button to abort the search
 		 */
+		isFocused: {
+			type: Boolean,
+			default: false,
+		},
+
 		isSearching: {
```

Comment:
We probably don't need this prop anymore, especially if it's calculated in parent based on this child's value